pkg/webhook/patch.go

```diff
@@ -206,6 +208,11 @@ func addGeneralConfigMaps(pod *corev1.Pod) []*patchOperation {
 	namesToMountPaths := config.FindGeneralConfigMaps(pod.Annotations)
 	for name, mountPath := range namesToMountPaths {
 		volumeName := name + "-vol"
+		if len(volumeName) > maxNameLength {
+			glog.V(2).Infof("ConfigMap volume name %s is too long. Truncating to length %d.", volumeName, maxNameLength)
```

Comment:
We can combine this and the log message below into one that's after the truncation and indicates the truncation and the result name.